src/Stripe.net/Services/Products/StripeProducNestedCreateOptions.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeProductNestedCreateOptions : INestedOptions
+    {
+        [JsonProperty("product[name]")]
+        public string Name { get; set; }
+
+        [JsonProperty("product[statement_descriptor]")]
+        public string StatementDescriptor { get; set; }
+    }
```

Comment:
We should support all options here and not just name and statement_descriptor. The API [docs](https://stripe.com/docs/api#create_plan-product) indicate we support `id` and `metadata` too.

Reply:
Good catch.